app/assets/stylesheets/ltags/UserSubscriptionTag.scss

```diff
@@ -1,85 +1,74 @@
+@import 'config/import';
+
 .ltag__user-subscription-tag {
-  .ltag__user-subscription-tag__container {
+  &__container {
+    border-radius: var(--radius);
+    border: 2px solid var(--body-color);
+    box-shadow: 4px 4px 0 var(--body-color);
+    box-sizing: border-box;
     display: flex;
     justify-content: center;
-    border: 2px solid black;
-    box-shadow: 4px 4px 0 black;
-  }
-
-  #profile-images {
-    text-align: center;
+    padding: var(--su-4);
 
-    img {
-      border-radius: 50%;
+    @media screen and (min-width: $breakpoint-m) {
+      padding: var(--su-6);
     }
   }
 
-  .ltag__user-subscription-tag__subscriber-profile-image {
-    display: none;
-  }
-
-  .ltag__user-subscription-tag__content {
-    margin: 1rem auto;
+  &__signed-in {
+    color: var(--base-70);
   }
 
-  .ltag__user-subscription-tag__cta-text {
-    text-align: center;
-  }
+  &__profile-images {
 
-  .ltag__user-subscription-tag__signed-out {
-    text-align: center;
-  }
+    &.signed-in {
+      margin: 0 auto var(--su-4);
+      position: relative;
+      height: 80px;
+      width: 80px;
+    }
 
-  .ltag__user-subscription-tag__response {
-    display: none;
-  }
+    &.signed-out {
+      height: auto;
+      width: auto;
+      text-align: center;
+      margin: 0 auto;
 
-  #sign-in-btn {
-    width: max-content;
-  }
+      .ltag__user-subscription-tag__author-profile-image {
+        margin: 0 auto var(--su-4);
+      }
+    }
 
-  .ltag__user-subscription-tag__signed-in {
-    text-align: center;
-    display: none;
+    img {
+      border-radius: 100%;
+    }
```

Comment:
If you use `.crayons-avatar` in the markup, it will automatically make avatars rounded. So you can drop this line.

Reply:
I added this because `.crayons-article__body img {border-radius: var(--radius)}` is over-writing the style of the rounded radius of `.crayons-avatar`. Have any thoughts on this hierarchy here?

Reply:
hmmm that's weird.. keep it as it is, i'll try to investigate that later 👍